recipe/meta.yaml

```
@@ -34,11 +34,11 @@ requirements:
     - patch  # [not win]
   host:
     - libprotobuf  # libprotobuf pins itself to x.x via run_exports
-    - libgsasl
+    - libgsasl 1.10
     - libntlm
     - libuuid
     - libxml2 2.10
```

```suggestion
    - libxml2
```
The cbc.yaml already has `libxml2 2.10`